File: 2021-SOPM/E1/E103AddingALibrary/app/src/main/java/com/example/e1_03_adding_a_library/MainActivity.java
package com.example.e1_03_adding_a_library;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.mackhartley.roundedprogressbar.RoundedProgressBar;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        RoundedProgressBar roundedProgressBar = findViewById(R.id.rounded_progress_bar);

        roundedProgressBar.setProgressPercentage(50.00, true);
    }
}

File: 2024-SOPM/PlantApp/app/src/main/java/com/example/plantapp/MainActivity.kt
package com.example.plantapp

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import com.example.plantapp.screens.MainScreen
import com.example.plantapp.ui.theme.PlantAppTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            PlantAppTheme {
                MainScreen()
            }
        }
    }
}